### ISSUE TITLE
Vmamba pretrain

### ISSUE THREAD

**Issue by kangben258** [MzeroMiko/VMamba#111]
I would like to ask if you have considered using some pre-training methods (for example, MAE, DINOV2, etc.) to pre-train Vmamba.

**Comment by MzeroMiko** [comment]
Yes. but due to the limitation of the resources we can use, we'll do it in the future, but not now.
